test/conftest.py

```diff
@@ -122,6 +130,22 @@ def teardown():
 
 
 def initialize_container_fixtures():
+    # create two networks to allow bond setups
+    # without intervening with the default network
+    for idx in range(2):
+        cmd = [
+            "docker",
+            "network",
+            "create",
+            "--driver=bridge",
+            f"infratest-net{idx}",
+            f"--subnet=172.19.{idx}.0/24",
+        ]
```

Comment:
I'm not happy with that, as it leaves the two networks remaining on the system executing the tests. I was not able to find a better place to put it and neither did I find a place to tidy up this completely. Something like a global fixture.

I'm not that familiar with pytest and all the possibilities to setup test environments.